docs/cadence/testing-framework.mdx

```diff
@@ -263,16 +738,31 @@ access(all) struct interface BlockchainBackend {
         account: Account,
         arguments: [AnyStruct]
     ): Error?
+
+    access(all) fun useConfiguration(_ configuration: Configuration)
+
+    access(all) fun logs(): [String]
+
+    access(all) fun serviceAccount(): Account
+
+    access(all) fun events(_ type: Type?): [AnyStruct]
+
+    access(all) fun reset(to height: UInt64)
+
+    access(all) fun moveTime(by delta: Fix64)
 }
+
 ```
 
 ### Creating a blockchain
 
-A new blockchain instance can be created using the `newEmulatorBlockchain` method.
-It returns a `Blockchain` which is backed by a new [Flow Emulator](../tools/toolchains/emulator) instance.
+A new blockchain instance can be created using the `Test.newEmulatorBlockchain` method.
+It returns a `Blockchain` which is backed by a new [Flow Emulator](../tooling/emulator) instance.
```

Comment:
I think this link here is causing the CI failure. Can you maybe double-check?

Reply:
My bad, I updated my fork, but forgot to pull the changes. I did so when some conflicts showed up, but this one probably sneaked in :sweat_smile: